addons/base_automation/views/base_automation_views.xml

```diff
@@ -53,15 +58,19 @@
                                     </div>
                                     <div class="text-muted" invisible="trigger != 'on_change'"><i class="fa fa-warning"/> Automation rules triggered by UI changes will be executed <em>every time</em> the watched fields change, <em>whether you save or not</em>.</div>
                                 </div>
-                                <label for="url" string="URL" invisible="trigger != 'on_webhook'"/>
-                                <div invisible="trigger != 'on_webhook'">
-                                    <field name="url" widget="CopyClipboardURL"  placeholder="URL will be created once the rule is saved."/>
-                                    <div class="alert alert-warning" role="status">
-                                        <strong><i class="fa fa-lock"/> Keep it secret, keep it safe.</strong>
-                                        <p>Your webhook URL contains a secret. Don't share it online or carelessly.</p>
```

Comment:
In my opinion that warning was necessary. Even a dev can forget that such data is supposed to stay private as much as possible